app/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -0,0 +1,17 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Looking to customize Bootstrap styles? Place SCSS variables
+// modifications *above* the Bootstrap import below. Learn more:
+// https://getbootstrap.com/docs/4.4/getting-started/theming/
+
+$grid-breakpoints: (
+  xs: 0,
+  sm: 480px,
+  md: 768px,
```

Comment:
I'm guessing there's no way to add the `min-height` of `481px` here to match FxA right? I thought that might be complex, if so I think it's totally fine to leave it out since mobile styles won't get much love on this project anyway.

Reply:
Correct - this variable is used as a sizing value beyond breakpoints, so a min-height wouldn't be applicable here. However, this doesn't preclude us from doing something custom if we want.

> mobile styles won't get much love on this project anyway.

This makes me sad. Would like to discuss this more at some point.